app/server/tests/api-deletePostTags.test.js

```diff
@@ -0,0 +1,72 @@
+const { AuthFunctions } = require('./functions/authFunctions');
+const { password, name, email } = require('./functions/randomizer');
+const { PostFunctions } = require('./functions/postFunctions');
+const community = require('./functions/communityFunctions');
+
+let auth = new AuthFunctions();
+let post = new PostFunctions();
+
+
+describe('Testing user\'s ability to DELETE Post Tags', () => {
+    let loginResponse;
+    let response;
+    let postResponse;
+    let userName = name.gen();
+    let userPassword = password.gen();
+    let communityName = name.gen();
+    let randomText = name.gen();
+    let tag1 = 'great';
+
+    beforeAll(async () => {
+        // Set up user
+        await auth.register(userName, email.gen(), userPassword);
+        loginResponse = await auth.login(userName, userPassword);
+
+        // Create some communities
+        await community.createCommunity(communityName, randomText, randomText, [{"tag": tag1, "count": 1}], loginResponse.body.token);
+
+        // Join communities
+        await community.joinCommunity(communityName, loginResponse.body.token);
+
+        // Create some posts
+        postResponse = await post.createPost(randomText, randomText, communityName, loginResponse.body.token);
+
+        // Tag that post
+        await post.createPostTags(postResponse.body._id, tag1, loginResponse.body.token);
+    });
+
+    afterAll(async ()=> {
+        await post.deleteAllPosts();
+        await community.deleteCommunity(communityName, loginResponse.body.token);
+        await auth.deleteUsers();
+    });
+
+    test('User can delete existing post tag.', async () => {
+        response = await post.deletePostTags(postResponse.body._id, loginResponse.body.token);
+        expect(response.status).toBe(204);
+        expect(response.body).not.toEqual(
```

Comment:
An unsuccessful tag would result in a 404, 403, or 400 status and does not include any json content.

Reply:
Good point. I'll add a GET in there to return the tag array and check the not-expected tag against that.